src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -253,6 +256,7 @@
     <EmbeddedResource Include="$(MsBuildThisFileDirectory)Resources\$(AssemblyName).rd.xml" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="System.Buffers" />
```

Comment:
You can only add this for netcoreapp as well as include the ValueStringBuilder since the converters are only included in netcoreapp.